bite-size-notes/week-2-d-Intro-To-Convex-Optimization.md

```diff
@@ -0,0 +1,143 @@
+# Intro to Convex Optimization
+
+#### Authors note 
+I tried to focus on getting down the more formal definitions for optimization and the notation for writing up problems. Usually, when I go back and try to remember a topic the notation and an explanation of some of the basics I may have forgotten help me to brush of the dust so to speak. I am also quite new to this topic, so some of the insights might seem surface level, but I am open to diving deeper into areas in the revision since I can already tell I've learned a lot just by writing these notes. 
+
+## Scope and Objective
+These notes will serve as a brief introduction to Convex Optimization within the field of Controls. Since this is also my introduction to optimization, it will cover more of the basics of convex optimization and what distinguishes it from other optimization problems.  
+
+## Introduction to optimization
+Optimization is a topic that is used in mathematics, computer science, engineering, and economics. It is a method of finding the best or optimal solution to a defined problem or system. Optimization problems can occur on an incredibly broad scale, going from optimizing everyday things like coffee consumption based on your relaxed constraints in your everyday life to optimizing aircraft performance based on 500 strict mechanical, aerodynamic, and financial constraints. Almost any system can be made into an optimization problem, and due to advancements in mathematical computing, most problems can be solved using a standard method and code base.
+
+In controls, optimization allows us not just to choose a feasible control input for our system, or even just a good control for our system, but to actually find the best control input for our system based on our constraints, desired behavior, and current state. This can be a powerful tool. However, there are some limiting factors. While the mathematical framework for setting up an optimization problem can be used to define most problems, actually solving a problem can yield very crude solutions. The key to solving an optimization problem simply and convincingly is to frame the problem using a convex objective function and convex subset[3]. The rest of these notes will explain what convex functions, subsets, and optimizations are and how to recognize and build systems that fit within them.   
+
+
+## Preliminaries  
+
+### Convexity 
+Mathematics defines a function as convex if a line segment can be drawn between any two distinct points, and the rest of the function stays below that line.  
```

Comment:
1. Change 'mathematics defines a function as convex' to 'A function is convex if and only if...'. One fix is word usage but the other is that this definition of convexity is necessary and sufficient i.e., we need "if and only if" instead of "if". Although to be fair this isn't a definition you have here but rather just a sentence so the informality (it not being  a definition) might be okay to just use if. Use your discretion.